### DISCUSSION TITLE
Auto-boot fw

### DISCUSSION THREAD

**Discussion by eduardokenj** [darthcloud/BlueRetro#59] [Q&A] [answered]
Does blueretro has a auto-boot fw? Because i press the boot botton ever time when power on the console

**Comment by darthcloud** [comment, marked as answer]
Yes, it start by itself. Just power on console.

**Reply by eduardokenj** [reply]
Or I made a mistake in programming the esp32? Because the fw not boot automatically, yes I flowed the guide to programming in windows

**Reply by darthcloud** [reply]
What kind of log you get if you simply plug your ESP32 to PC via USB:
https://github.com/darthcloud/BlueRetro/wiki/Getting-BlueRetro-debug-logs-via-Serial-port-Windows-10

**Reply by eduardokenj** [reply]
![image](https://user-images.githubusercontent.com/7385867/129621643-2f2cab0b-5f0c-4b56-b285-030fdd261fd6.png)
i got these garbage bytes in my terminal, i hooked the TX pin to RX pin in my FTDI serial conv.
![image](https://user-images.githubusercontent.com/7385867/129621934-fcdbb1f8-2467-479a-9135-5dcba2cb1c01.png)
but after presse the reset swt, the fw boot

**Reply by darthcloud** [reply]
What kind of board you use? Your board might be faulty.

**Reply by eduardokenj** [reply]
An a esp32 dev board like this:
![20210816_170852](https://user-images.githubusercontent.com/7385867/129623529-b6485ce8-8edc-4f62-8202-108051d57bb5.jpg)

**Reply by eduardokenj** [reply]
It's programming normally, but in operation itself is buggy

**Reply by darthcloud** [reply]
Try avoid breadboard that often make trouble, if making a PSX adapter, just connect 7 wire directly. from psx plug to esp32.
